fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/RecordConstructorValue.java

```diff
@@ -154,7 +155,7 @@ private DynamicMessage.Builder newMessageBuilderForType(@Nonnull TypeRepository
      */
     @Nullable
     @SuppressWarnings("PMD.CompareObjectsWithEquals")
-    private Object deepCopyIfNeeded(@Nonnull TypeRepository typeRepository,
+    public static Object deepCopyIfNeeded(@Nonnull TypeRepository typeRepository,
```

Comment:
Please add [`@VisibleForTesting`](https://guava.dev/releases/19.0/api/docs/com/google/common/annotations/VisibleForTesting.html) because I think the only reason it is made `public` is to enable unit-testing it, right?

Reply:
done